crates/llama-cpp-server/src/lib.rs

```diff
@@ -202,6 +202,32 @@ pub async fn create_completion(
     (stream, prompt_info)
 }
 
+pub async fn create_completion_and_chat(
```

Comment:
This seems to still start two llama-server subprocesses, not actually reusing the process?

Reply:
still working on it, I first changed the outside logic, it's nearly done.
But it's too late last night, so I created this draft, will fix it today.